CHANGELOG.md

```diff
@@ -1,6 +1,30 @@
 Magento Functional Testing Framework Changelog
 ================================================
 
+2.4.0
+-----
+### Enhancements
+* Maintainability
+    * Added new `mftf static-checks` command to run existing static checks against the attached test codebase
+        * See DevDocs for details
+    * Added new `mftf generate:docs` command that generates documentation about attached test codebase
+        * See DevDocs for details
+* Traceability
+    * Allure reports for tests now contain collapsible sections for `actionGroup`s used in execution.
+
+### Fixes
+* Fixed an issue where `magentoCli` would treat `argument="0"` as a null value.
+* Fixed an issue where `amOnPage` and `waitForPwaElementVisible` would not utilize the `timeout` attribute correctly.
+* Fixed an issue where invalid XML characters would cause Allure to throw an exception without a resulting report.
+* Fixed `codeception.dist.yml` configuration for keeping previous test run results.
+* PHP Notices are no longer thrown then XML is missing non-necessary attributes.
+* Removed unusable `fillSecretField` action from schema.
+
+### GitHub Issues/Pull requests:
+* [#338](https://github.com/magento/magento2-functional-testing-framework/pull/338) -- Return exit codes of process started by 'run:test', 'run:group' or 'run:failed' command
+* [#333](https://github.com/magento/magento2-functional-testing-framework/pull/333) -- Added Nginx specific settings to getting started doc
+* [#318](https://github.com/magento/magento2-functional-testing-framework/pull/318) -- Reduce cyclomatic complexity in Problem Methods
+
```

Comment:
And MQE-1521 for executeInSelenium() generation fix?

Reply:
Will do 👍